### ISSUE TITLE
Add a link to activate bomb with Competitive Mode

### ISSUE THREAD

**Issue by tepel-chen** [samfundev/KTANE-Bombs#91]
For example:

http://localhost:8085/startMission?missionID=mod_100LevelsOfDefusalMissions_leve100&seed=-1

activates Level 100 if Competitive Mode Mod is installed

**Comment by Luminoscity** [comment]
I have no idea what this request is about. You want the challenge bomb website to access a mod on a user's personal computer and launch the game?

**Comment by tepel-chen** [comment]
Yes, that's right

**Comment by Luminoscity** [comment]
I don't think we are okay with the website doing that, accessing and activating a Steam workshop mod that may or may be installed on a user's personal computer and is out of our control.
